### ISSUE TITLE
No 'Update' button

### ISSUE THREAD

**Issue by Brick36** [envato/wp-envato-market#60]
Hi,

I've installed the Envato plug-in, verified with the token, and it recognises what theme I have installed. 

 need to update my theme, but I don't have an 'update' button - only an 'install' one (for the theme I already have). 

I tried clicking 'install' just in case that works, but it doesn't. I get an error message telling me I already have that theme installed. 

I'm using 4.5.3 version of WordPress and 'Fmagazine' theme - I need to update to version 5 of this. Please can you let me know how to resolve this?

Thanks

**Comment by cubewebsolutions** [comment]
I've asked the same question in here, but no one has bothered to respond.

**Comment by IgorCode** [comment]
See this comment https://github.com/envato/wp-envato-market/issues/54#issuecomment-230074130

**Comment by valendesigns** [comment]
This is likely because the theme author or file name has changed and does not match anymore so the plugin only thinks you can install. But when it tries to install it fails since it does already exist. I would need to know if there is an obvious reason that it's not matching and account for that in the matching check. I just need more info on what's changed.

**Comment by Brick36** [comment]
Thanks for getting back to me valendesigns. I have no idea what's changed - the theme is still called Fmagazine. The author is Themeruby

I don't know what other info you would need for the matching check? If you let me know I can send over. Keen to get this resolved.

Thanks

**Comment by DariDesignagent** [comment]
I changed the theme's name midway review process. The new Envato policy is - they don't change the URLs, when the item name was changed. The theme's name and theme's URL don't match anymore.

So now the Envato Market plugin does not work for any of my buyers. There is only the 'Install' button, no 'Update' button.

**Comment by dtbaker** [comment]
There have been some recent API changes. Hey please let us know if this is still an issue and we can look into it further.

**Comment by Evelyn7** [comment]
Hi+
I downloaded the zip file from your website on 02/26/18, then removed the old plugin and installed the new zip file via the back-end Plugins > add new > upload. I then activated - and it still did not work. It did show an "update" button/link, however, when I clicked that nothing happened. 

WP v 4.9.4
Envato Plugin v. 1.0.0-RC2

So, is the zip file on your website the "new" (fixed) version of the plugin? If so, why is it not working as expected? If not, where do I find the correct version?
Thanks!
